Update _index.en.md (#24)
go-skynet/localai-website: content/getting_started/_index.en.md

Requirement: nvidia-container-toolkit (installation instructions [1](https://www.server-world.info/en/note?os=Ubuntu_22.04&p=nvidia&f=2) [2](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html))

> To get started using Cuda GPU Update the docker file.

Reviewer: ```suggestion
To get started using CUDA GPU with `docker-compose`, see the following example:
```

```
version: '3.6'

services:
  api:
    deploy:
      resources:
        reservations:
          devices:
            - driver: nvidia
              count: 1
              capabilities: [gpu]
    image: quay.io/go-skynet/local-ai:master-cublas-cuda12

>     build:

Reviewer: ```suggestion
```

>       context: .

Reviewer: ```suggestion
```

>       dockerfile: Dockerfile

Reviewer: ```suggestion
```

    tty: true # enable colorized logs

>     restart: always # should this be on-failure ?

Reviewer: Better to not leave open questions in the example
```suggestion
    restart: always # or also on-failure
```

    ports:
      - 8080:8080

>     env_file:

Reviewer: ```suggestion
```

>       - .env

Reviewer: ```suggestion
```

    restart: always # or also on-failure
    ports:
      - 8080:8080

>     env_file:

Author: This is needed to load the users env file...?

    ports:
      - 8080:8080
    env_file:

>       - .env

Author: this is needed to load the env file...
